java/lang/Integer/IntegerTest.cpp

```diff
@@ -327,8 +335,15 @@ TEST (JavaLang, IntegerDoubleValue) {
     double realResult = positiveInteger.doubleValue();
     ASSERT_TRUE(expectedResult == realResult);
 
+    // Make a comparison between expected realResult and the real result
     Integer negativeInteger = -123456789;
     expectedResult = -123456789.0000000;
     realResult = negativeInteger.doubleValue();
     ASSERT_TRUE(expectedResult == realResult);
+
+    // Make a comparison between the not expected realResult and the real result
+    Integer notExpectedInteger = -12345;

```

Comment:
The same answer:

for that code : ASSERT_FALSE(notExpectedResult == realResult);  
I want to test the case : the realResult and the expectedResult are different. So I call it notExpectedInteger.